Week2/src/clients/getMessageStatus.js

```diff
@@ -0,0 +1,8 @@
+import { Message, BackUpMessage } from "../models/message.js";
```

Comment:
You are importing `BackUpMessage` but you are not using it 🤔

Reply:
Yeah, that's because I wanted to implement the back-up db also for messages but I ran out of time. I will these days.